use/install.rst

```diff
@@ -57,35 +57,31 @@ Python will usually come by installed by default in your distribution. If not,
 grab the appropriate packages from the distribution's repository, or download
 it from https://python.org.
 
-Install Limnoria
-----------------
+Install pipx
+------------
 
-In the next section of this guide we will use `pip`_, which is a generic
-way of installing Python software.
+This is a tool for installing Python applications. Follow the `install instructions`_ for your operating system. 
 
-.. _pip: https://pip.readthedocs.org/en/latest/installing.html#install-pip
+.. _install instructions: https://pipx.pypa.io/latest/#install-pipx
 
-First, you need to setup a virtualenv, which is the directory where we
-will install Limnoria's code (but not its configuration)::
+For example, on a system with the apt package manager::
 
-    mkdir -p $HOME/.venvs/
-    python3 -m venv $HOME/.venvs/limnoria  # creates a virtualenv at the given path
-    . $HOME/.venvs/limnoria/bin/activate   # enables the virtualenv in the current shell
+    sudo apt update
+    sudo apt install pipx
+    pipx ensurepath
```

Comment:
Could you swap them? The `apt` case should cover most users (especially the beginner) so it should be first. Some thing like this: "If you use an APT-based distribution (eg. Debian or Ubuntu): [commands] otherwise, follow the `install instructions`_ for your operating system."